src/Pickles/Pickles.DocumentationBuilders.Json.UnitTests/FormattingAFeature.feature

```diff
@@ -153,91 +153,201 @@ Scenario: A feature with a table
                           "DataRows": [
                             [
                               "Chapter 1",
-                              "1"
+                              "1",
```

Comment:
I took the liberty of improving the formatting here, and pushed that change to your repository.

Reply:
@dirkrombauts : that's my fault - i forgot to invite you to my repo. I just have it done so you could push your changes whenever you want now :)

Reply:
thanks, I just pushed my changes